sdcm/cluster_k8s/eks.py

```diff
@@ -334,6 +345,41 @@ def deploy(self):
         self.create_eks_cluster()
         LOGGER.info("Patch kubectl config")
         self.patch_kubectl_config()
+        LOGGER.info("Create storage class")
+        self.create_ebs_storge_class()
+
+    def create_ebs_storge_class(self):
```

Comment:
I'm failing to understand how create a new storage class is to with cpu/memory resources
or you referring to the resources used by ebs-csi-driver ?

Reply:
Yes, `ebs-csi-driver` has resources requests for CPU and memory and for now it's appearance increases the probability that the requests for scylla pods may not be satisfied because they are calculated without regards for requests of the new pods.

Reply:
csi driver add 3 pods with such configuration for each node:
```
#     resources:
#       limits:
#         cpu: 100m
#         memory: 256Mi
#       requests:
#         cpu: 10m
#         memory: 40Mi
```

you think we should make the `cpu: 0.51` more cause of this new 30m requests ?

Reply:
o.k. changed